dev/details.html

```diff
@@ -5,27 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Details</title>
-    <script type="module" src="./common.js"></script>
-
-    <script type="module">
-      import '@vaadin/details';
-      import '@vaadin/tooltip';
-      // import '@vaadin/details/theme/material/vaadin-details.js';
-      // import '@vaadin/tooltip/theme/material/vaadin-tooltip.js';
-    </script>
   </head>
 
   <body>
     <vaadin-details summary="Expandable Details">
       <div>Toggle using mouse, Enter and Space keys.</div>
     </vaadin-details>
 
-    <!--
-    <vaadin-details>
-      <vaadin-details-summary slot="summary">Expandable Details</vaadin-details-summary>
-      <div>Toggle using mouse, Enter and Space keys.</div>
-      <vaadin-tooltip slot="tooltip" text="Click to toggle"></vaadin-tooltip>
-    </vaadin-details>
-    -->
+    <script type="module">
+      if (window.useBaseStyles) {
+        import('@vaadin/details/src/vaadin-details.js');
```

Comment:
How about providing an option to specify which theme to use when running the dev pages? For example: `--theme={base|lumo|material}` which would automatically rewrite all URLs on the dev pages accordingly.

Reply:
That way, we could just import the unstyled components, and the dev server would automatically inject the Lumo or Material imports when needed.

Reply:
We'll remove Material theme soon anyway. Maybe this option can be added later once we start adding Aura.